README.md

```diff
@@ -40,7 +40,7 @@ Add to `composer.json` in your project to `require` section:
 ```
 
 Run command:
-`php composer.phar install`
+`php composer.phar update`
```

Comment:
Imo you should rather replace the whole installation instructions with the following command:

```
composer require suncat/mobile-detect-bundle
```